session/Concept.proto

```diff
@@ -151,56 +122,85 @@ message Method {
-            SchemaConcept.Subs.Iter.Res schemaConcept_subs_iter_res = 206;
-
-            // Role iterator responses
-            Role.Relations.Iter.Res role_relations_iter_res = 401;
-            Role.Players.Iter.Res role_players_iter_res = 402;
-
-            // Type iterator responses
-            Type.Instances.Iter.Res type_instances_iter_res = 502;
-            Type.Keys.Iter.Res type_keys_iter_res = 503;
-            Type.Attributes.Iter.Res type_attributes_iter_res = 504;
-            Type.Playing.Iter.Res type_playing_iter_res = 505;
-
-            // RelationType iterator responses
-            RelationType.Roles.Iter.Res relationType_roles_iter_res = 701;
-
-            // Thing iterator responses
-            Thing.Keys.Iter.Res thing_keys_iter_res = 902;
-            Thing.Attributes.Iter.Res thing_attributes_iter_res = 903;
-            Thing.Relations.Iter.Res thing_relations_iter_res = 904;
-            Thing.Roles.Iter.Res thing_roles_iter_res = 905;
-
-            // Relation iterator responses
-            Relation.RolePlayersMap.Iter.Res relation_rolePlayersMap_iter_res = 1000;
-            Relation.RolePlayers.Iter.Res relation_rolePlayers_iter_res = 1001;
+        message Req {
+            oneof iter_req {
+                // SchemaConcept iterator requests
+                SchemaConcept.Sups.Iter.Req schemaConcept_sups_iter_req = 205;
+                SchemaConcept.Subs.Iter.Req schemaConcept_subs_iter_req = 206;
+
+                // Role iterator requests
+                Role.Relations.Iter.Req role_relations_iter_req = 401;
+                Role.Players.Iter.Req role_players_iter_req = 402;
+
+                // Type iterator requests
+                Type.Instances.Iter.Req type_instances_iter_req = 502;
+                Type.Keys.Iter.Req type_keys_iter_req = 503;
+                Type.Attributes.Iter.Req type_attributes_iter_req = 504;
+                Type.Playing.Iter.Req type_playing_iter_req = 505;
+
+                // RelationType iterator requests
+                RelationType.Roles.Iter.Req relationType_roles_iter_req = 701;
+
+                // Thing iterator requests
+                Thing.Keys.Iter.Req thing_keys_iter_req = 902;
+                Thing.Attributes.Iter.Req thing_attributes_iter_req = 903;
+                Thing.Relations.Iter.Req thing_relations_iter_req = 904;
+                Thing.Roles.Iter.Req thing_roles_iter_req = 905;
+
+                // Relation iterator responses
+                Relation.RolePlayersMap.Iter.Req relation_rolePlayersMap_iter_req = 1000;
+                Relation.RolePlayers.Iter.Req relation_rolePlayers_iter_req = 1001;
+
+                // Attribute iterator requests
+                Attribute.Owners.Iter.Req attribute_owners_iter_req = 1101;
+            }
+        }
 
-            // Attribute iterator responses
-            Attribute.Owners.Iter.Res attribute_owners_iter_res = 1101;
+        message Res {
+            oneof iter_res {
```

Comment:
let's rename `iter_res` to `res` to be consistent with other `oneof` variable names (being just the lowercasing of their owning `message`).